Sorting/recursive_bubbleSort.cpp

```diff
@@ -0,0 +1,40 @@
+#include <bits/stdc++.h>
+using namespace std;
+ 
+
+void bubbleSort(int arr[], int n)   //bubble sort FUNCTION
+{
+    // Base case
+    if (n == 1)
+        return;
+ 
+    
+    for (int i=0; i<n-1; i++)
+        if (arr[i] > arr[i+1])
```

Comment:
IMHO you can improve this algorithm just looking if there is a swap, if there is not the bubble sort could end in that iteration.

Reply:
where would be the swap,in the for loop?